projects/los-angeles-metro/README.md

```diff
@@ -0,0 +1,35 @@
+
+# Portland Metro Area
```

Comment:
There are a few references to Portland in the readme, also one more in the next sentence.